docs/io/unified.rst

```diff
@@ -318,6 +316,15 @@ the keywords ``DATE``, ``DATE-*`` (ISO-8601 datetime strings) and the ``MJD-*``
 For more details regarding the FITS time paper and the implementation,
```

Comment:
@AustereCuriosity - given the comments in the main thread, maybe the last part should be
```
`~astropy.table.QTable` (but otherwise, the classes behave identically).
```